libsignal-service-actix/src/push_service.rs

```diff
@@ -47,7 +47,7 @@ impl AwcPushService {
         credentials_override: HttpAuthOverride,
     ) -> Result<ClientRequest, ServiceError> {
         let url = self.cfg.base_url(endpoint).join(path.as_ref())?;
-        log::debug!("HTTP request {} {}", method, url);
+        tracing::debug!("HTTP request {} {}", method, url);
```

Comment:
We can definitely merge this as is, but if you want to use the feature of `tracing` that adds support for structured logging, the syntax changes quite a bit.

```suggestion
        tracing::debug!(%url, %method, "HTTP request");
```

one can also rename fields in the logging statements, like so:

`tracing::debug!(http_url =% url, "HTTP request")`

You need to prefix the fields with `%` to use `fmt::Display` or `?` to use `fmt::Debug` when the type doesn't implement `tracing::Value`.

Reply:
... that's quite a QoL improvement and would remove the need for #129 / #53 !

Reply:
I changed quite some calls here. I suggest we clean up while we go, this is looking very pretty already on the console :)